docs/Report.md

```diff
@@ -158,19 +163,26 @@ This section will describe what features and implementation weren't completed an
+
+Three issues regarding the old retired Chirp CLI application is closed, but not implemented. The issues can be seen in the far right column, and is:
+- Adding automatic deployment from GitHub to the host service containing the web api.
+- Changing the application to use the database on the web service instead of the local hosted database
+- Ensuring that the test coverage are adequate after refactoring our wep api
 
 ## Issue creation
 ![Fig.XX Flow of issues](Images/teamwork.png){width=60%}
 <br>
-This activity diagram shows the flow of our work process. At first, the new requirements are read and understood, and then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a day's worth of work. If a formulation gets accepted by the group it gets posted on the issue board on Github. A developer assigns themselves to an issue to let others know what they are working on. When the developer feels like they've implemented the feature adequately they commit and create a pull request. When a pull request is posted two reviewers from the group are needed to further merge it to main and deploy. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until two reviewers accept the changes and then the code can be merged with main. 
+This activity diagram shows the flow of our work process. At first, the new requirements are read and understood, and then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a day's worth of work. If a formulation gets accepted by the group it gets posted on the issue board on Github. A developer assigns themselves to an issue to let others know what they are working on. When the developer feels like they've implemented the feature adequately, that is the acceptance criterias are met, they commit and create a pull request. When a pull request is posted a minimum og two reviewers from the group are needed to accept thee pull request, before merging it into main and deploy to Azure. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until the same reviewer accepts the changes and then the code can be merged into main. 
 
+The group had a gentleman agreement, if not all members of the group has reviewed a pull request, one must wait a 24 hours before merging into main, even if two reviewers has accepted the pull request. This is to allow every member of the group to have a chance of reviewing the pull request, but still not bringing development to a stall. This offered a balance between keeping everyone "in the loop" and being able to implement changes fairly quickly, that we found to be working fairly well.
```

Comment:
I honestly disagree about putting this in as I feel like it hasn't been complied with almost at all. Additionally one can easily see pull requests that have been closed so there is no need to keep them open "to keep everyone in the loop"

Reply:
otherwise if the group waat's to keep it and it's just me i would prefer removing "gentleman agreemnt" as I feel that doesnt fit into a report very well